test/integration/cleanliness_test.rb

```diff
@@ -43,9 +43,9 @@ def check_content(files)
     end
   end
 
-  it "does not have actions on base controller" do
+  it "does not have public actions on base controller" do
     found = ApplicationController.action_methods.to_a
-    found.reject { |a| a =~ /^(_conditional_callback_around_|_callback_before_)/ }
+    found.reject! { |a| a =~ /^(_conditional_callback_around_|_callback_before_|api_)/ }
```

Comment:
but that's a private method ... and static ?
maybe the class_attribute needs `instance_accessor: false`

On Tue, Aug 9, 2016 at 11:30 AM, Adam Panzer <notifications@github.com>
wrote:

> In test/integration/cleanliness_test.rb
> <https://github.com/zendesk/samson/pull/1163#discussion_r74117552>:
>
> >      found = ApplicationController.action_methods.to_a
> > -    found.reject { |a| a =~ /^(_conditional_callback_around_|_callback_before_)/ }
> > +    found.reject! { |a| a =~ /^(_conditional_callback_around_|_callback_before_|api_)/ }
>
> It is public because api_accessible!(true)/(false) works.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1163/files/ec9176e57ff6e9773ea877bdef9761f323850b46#r74117552>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ-7GLUA2c4XXGdAnyywJ9pGolzvZks5qeMdggaJpZM4JVi_g>
> .
>